util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -155,24 +155,19 @@
   public static CredentialFactory getCredentialFactory(
       Configuration config, String... keyPrefixesVararg) {
     List<String> keyPrefixes = getConfigKeyPrefixes(keyPrefixesVararg);
+    RedactedString saEmailSuffix =
+        SERVICE_ACCOUNT_EMAIL_SUFFIX.withPrefixes(keyPrefixes).getPassword(config);
     CredentialOptions credentialOptions =
         CredentialOptions.builder()
             .setServiceAccountEnabled(
                 ENABLE_SERVICE_ACCOUNTS_SUFFIX
                     .withPrefixes(keyPrefixes)
                     .get(config, config::getBoolean))
             .setServiceAccountPrivateKeyId(
-                RedactedString.create(
-                    SERVICE_ACCOUNT_PRIVATE_KEY_ID_SUFFIX
-                        .withPrefixes(keyPrefixes)
-                        .getPassword(config)))
+                SERVICE_ACCOUNT_PRIVATE_KEY_ID_SUFFIX.withPrefixes(keyPrefixes).getPassword(config))
```

Comment:
Should we use `RedactedString` here as well ?

Reply:
It's used automatically - `getPassword` returns `RedactedString` now.